document/core/binary/modules.rst

```diff
@@ -327,13 +328,16 @@ It decodes into a vector of :ref:`element segments <syntax-elem>` that represent
      \X{seg}^\ast{:}\Bsection_9(\Bvec(\Belem)) &\Rightarrow& \X{seg} \\
    \production{element segment} & \Belem &::=&
      \hex{00}~~e{:}\Bexpr~~y^\ast{:}\Bvec(\Bfuncidx)
-       &\Rightarrow& \{ \ETABLE~0, \EOFFSET~e, \EINIT~y^\ast \} \\
+       &\Rightarrow& \{ \ETABLE~0, \EOFFSET~e, \EINIT~(\EREFFUNC~y)^\ast \} \\
    \production{element segment} & \Belem &::=&
-     \hex{01}~~y^\ast{:}\Bvec(\Bfuncidx)
+     \hex{01}~~y^\ast{:}\Bvec(\Belement)
        &\Rightarrow& \{ \EINIT~y^\ast \} \\
    \production{element segment} & \Belem &::=&
      \hex{02}~~x{:}\Btableidx~~e{:}\Bexpr~~y^\ast{:}\Bvec(\Bfuncidx)
-       &\Rightarrow& \{ \ETABLE~x, \EOFFSET~e, \EINIT~y^\ast \} \\
+       &\Rightarrow& \{ \ETABLE~x, \EOFFSET~e, \EINIT~(\EREFFUNC~y)^\ast \} \\
+   \production{element} & \Belement &::=&
```

Comment:
Better name for this? currently we use `elem` to mean "element segment", and I'm not sure it's worth changing that everywhere.

Reply:
I used `elemexpr`. Rebasing seems like it might be more confusing, to me unfortunately...